app/constants/index.js

```diff
@@ -17,3 +17,5 @@ export const newsBaseURL = 'https://www.lahmacun.hu/wp-json/wp/v2/posts?categori
 export const newsURL = 'https://www.lahmacun.hu/wp-json/wp/v2/posts?categories=5&per_page=12'
 
 export const aboutUsURL = 'https://www.lahmacun.hu/wp-json/wp/v2/pages/228'
+export const donateURL = 'https://www.lahmacun.hu/wp-json/wp/v2/pages/2'
+export const favouriteRadiosURL = 'https://www.lahmacun.hu/wp-json/wp/v2/pages/2875'
```

Comment:
@baxgas I guess it would be good to centrally configure the WP site so that we need to change it only once when we redirect the domain.

Reply:
@gammaw – as for the above, all the content comes from our WP site, if it is changed there it will update here also
nice catch about the endpoints, I made them modular